### ISSUE TITLE
crash after start, windows, 4.1.2

### ISSUE THREAD

**Issue by pomazanbohdan** [lumehq/lume#231]
same as https://github.com/lumehq/lume/issues/223

**Comment by reyamir** [comment]
Oh, seems strange.
Can you help me check is the webview2 (https://developer.microsoft.com/en-us/microsoft-edge/webview2/?form=MA13LH#download-section) installed on your system yet?

**Comment by pomazanbohdan** [comment]
i use win 11 and have webview2 in default.
Installed Version:  128.0.2739.54

**Comment by klendazu** [comment]
Same to me, the app not start.

**Comment by reyamir** [comment]
Yes, I've faced same issue, but didn't find the root cause yet. I think there are some issues with the window effect (mica). I will try to generate a build without any window effect
